doc/changes.rst

```diff
@@ -1,6 +1,113 @@
+* Apply standardized formatting conventions (`#775 <https://github.com/GenericMappingTools/pygmt/pull/775>`__) across most documentation pages
+* Drop Python 3.6 support (`#699 <https://github.com/GenericMappingTools/pygmt/pull/699>`__) so PyGMT now requires Python 3.7 or newer
+
+New Features
+
+* Wrap grd2cpt (`#803 <https://github.com/GenericMappingTools/pygmt/pull/803>`__)
+* Let Figure.text() support record-by-record transparency (`#716 <https://github.com/GenericMappingTools/pygmt/pull/716>`__)
+* Provide basic support for FreeBSD (#700, #878)
+
+Enhancements
+
+* Let load_earth_relief() support the 'region' parameter for all resolutions (`#873 <https://github.com/GenericMappingTools/pygmt/pull/873>`__)
+* Improve how PyGMT finds the GMT library (`#702 <https://github.com/GenericMappingTools/pygmt/pull/702>`__)
+* Add common alias panel (-c) to all plotting functions (`#853 <https://github.com/GenericMappingTools/pygmt/pull/853>`__)
+* Add aliases dcw (`#765 <https://github.com/GenericMappingTools/pygmt/pull/765>`__) and lakes (`#781 <https://github.com/GenericMappingTools/pygmt/pull/781>`__) to Figure.coast
+* Add alias shading to Figure.colorbar (`#752 <https://github.com/GenericMappingTools/pygmt/pull/752>`__)
+* Add alias annotation (A) to Figure.contour (`#883 <https://github.com/GenericMappingTools/pygmt/pull/883>`__)
+* Wrap Figure.grdinfo() aliases (`#799 <https://github.com/GenericMappingTools/pygmt/pull/799>`__)
+* Add aliases frame and cmap to Figure.colorbar (`#709 <https://github.com/GenericMappingTools/pygmt/pull/709>`__)
+* Add alias frame to Figure.grdview (`#707 <https://github.com/GenericMappingTools/pygmt/pull/707>`__)
+* Improve the error message when PyGMT fails to load the GMT library (`#814 <https://github.com/GenericMappingTools/pygmt/pull/814>`__)
+* Add GMTInvalidInput error to Figure.coast (`#787 <https://github.com/GenericMappingTools/pygmt/pull/787>`__)
+
+Documentation
+
+* Add authorship policy (`#726 <https://github.com/GenericMappingTools/pygmt/pull/726>`__)
+* Update PyGMT development installation instructions (`#865 <https://github.com/GenericMappingTools/pygmt/pull/865>`__)
+* Add a tutorial for adding a map title (`#720 <https://github.com/GenericMappingTools/pygmt/pull/720>`__)
+* Add a tutorial for plotting Earth relief (`#712 <https://github.com/GenericMappingTools/pygmt/pull/712>`__)
+* Add a tutorial for 3D perspective image (`#743 <https://github.com/GenericMappingTools/pygmt/pull/743>`__)
+* Add a tutorial for contour maps (`#705 <https://github.com/GenericMappingTools/pygmt/pull/705>`__)
+* Add a tutorial for plotting lines
+* Add a turorial for the region argument (`#800 <https://github.com/GenericMappingTools/pygmt/pull/800>`__)
+* Add a gallery example for datetime inputs (`#779 <https://github.com/GenericMappingTools/pygmt/pull/779>`__)
+* Add a gallery example for Figure.logo() (`#823 <https://github.com/GenericMappingTools/pygmt/pull/823>`__)
```

Comment:
```suggestion
* Add a gallery example for Figure.logo (`#823 <https://github.com/GenericMappingTools/pygmt/pull/823>`__)
```

Can you remove all the parentheses as mentioned in https://github.com/GenericMappingTools/pygmt/pull/887#discussion_r576008258?

Reply:
Fixed it in 98097bc